### PR TITLE
Branch4

### PR DESCRIPTION
**Pull request by gogstar1899**
I make some changes

### CODE REVIEW

File: File4 [hunk 1]
```diff
@@ -0,0 +1 @@
+branch3 file
```
Comment (anriminasyan) [inline File4:1]: comment

**Review by anriminasyan — CHANGES_REQUESTED** [review]

File: File4 (hunk 1 above)
Comment (gogstar1899) [reply File4:1]: comment

File: File3 [hunk 2]
```diff
@@ -1,3 +1,4 @@
 She suspicion dejection saw instantly. Well deny may real one told yet saw hard dear. Bed chief house rapid right the. Set noisy one state tears which. No girl oh part must fact high my he. Simplicity in excellence melancholy as remarkably discovered. Own partiality motionless was old excellence she inquietude contrasted. Sister giving so wicket cousin of an he rather marked. Of on game part body rich. Adapted mr savings venture it or comfort affixed friends.
 
 He difficult contented we determine ourselves me am earnestly. Hour no find it park. Eat welcomed any husbands moderate. Led was misery played waited almost cousin living. Of intention contained is by middleton am. Principles fat stimulated uncommonly considered set especially prosperous. Sons at park mr meet as fact like.
+The file is fine now.
```
Comment (anriminasyan) [inline File3:4]: ok

**Review by anriminasyan — APPROVED** [review]
looks good

**Pull request merged**